src/architecture/system_model/sys_model_task.cpp

```diff
@@ -37,7 +37,7 @@ SysModelTask::SysModelTask()
  Note that the only required argument is InputPeriod.
  @param InputPeriod The amount of nanoseconds between calls to this Task.
  @param InputDelay How long to delay the input by in nanoseconds
- @param FirstStartTime The offset in nanoseconds in a given frame to start the Task with.
+ @param FirstStartTime The amount of time in nanoseconds to hold a task dormant before starting.  After this time the task is executed at integer amounts of InputPeriod again
```

Comment:
:large_orange_diamond: This line runs over the desired 120 character line length. It can be broken.

Reply:
Done